### PR TITLE
Interaction models API: challenge/response

### PR DESCRIPTION
**Pull request by thomas-fossati**

### CODE REVIEW

File: docs/api/README.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+  Host: veraison.example
+  Accept: application/rats-challenge-response-session+json
+  Content-Type: application/psa-attestation-token
+
+  .....
+
+<< Response:
+  HTTP/1.1 202 Accepted
+  Content-format: application/rats-challenge-response-session+json
+
```
Comment (SimonFrost-Arm) [inline docs/api/README.md:44]: This response seems to be lacking a Location to be used in the subsequent poll?

Comment (SimonFrost-Arm) [reply docs/api/README.md:44]: Ah, wait - I didn't spot the sequence correctly: POST newSession and then POST to session as required. Might be worth adding a couple of lines about the interaction model to the top of the readme for anyone else reading this before a morning coffee.

**Review by SimonFrost-Arm — CHANGES_REQUESTED** [review]
All fine, though see comment about adding a couple of lines of intro to the readme outlining the model, being a clarification for someone coming to this fresh.

**Review by setrofim — APPROVED** [review]

**Review by yogeshbdeshpande — APPROVED** [review]

File: docs/api/README.md (hunk 1 above)
Comment (thomas-fossati) [reply docs/api/README.md:85]: Cool, done in caca362.  The pic came out a bit bizarre though -- the UML client  puppet looks like UFO Robot Grendizer firing HTTP requests like photonics rays!

**Review by henkbirkholz — APPROVED** [review]
+1

**Review by SimonFrost-Arm — APPROVED** [review]

**Pull request merged**